.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
               git config user.email accounts@plot.ly
               git add *
               git commit -m "build of https://github.com/plotly/plotlyjs.jl-docs/commit/${CIRCLE_SHA1}"
-              git push --force git@github.com:plotly/plotlyjs.jl-docs.git master:built_ipynb
+              git push --force git@github.com:plotly/plotlyjs.jl-docs.git master:built
```

Comment:
so I think the `git push` a few lines above is what I need... I don't need the IPYNB files, but if you want to keep them in `built_ipynb` that's fine with me.

To fully resolve #10 I'll need the HTML file names to start with a date too though.

Reply:
Ah I see you did that already :)  https://github.com/plotly/plotlyjs.jl-docs/commit/89c2e494d9b384bb73dc512693bbf917cb3c0105

Reply:
yep -- already have the date :)

Also, while you were OOO I ran into some bugs in the jupyter pipeline. I have replaced it with a custom (<200 LoC) solution in pure julia. There are no more notebooks. 

Ha so I think that means I should delete this whole ipynb section of CI config and we shoudl be good to go?

Reply:
sure yeah delete the Ipynb section, and then I'm logging some more notes in #10 for small things that need adjustment before the output can be fed into the next part of the pipeline. We're super close :) Thanks!